me.njean.shootinpactris.json

```diff
@@ -0,0 +1,89 @@
+{
```

Comment:
The manifest file should be named `me.njean.shootingpactris.json`

Reply:
Oups! Typo now fixed.